composer.json

```diff
@@ -275,6 +277,11 @@
         },
         "runtime": {
             "dotenv_path": "project-base/app/.env"
+        },
+        "patches": {
+            "overblog/graphql-bundle": {
+                "Change default validator domain": "project-base/app/patch/overblog-validator.patch"
```

Comment:
should not be the patch in fe api package?

Reply:
Just the patch file, or the patch file and composer.json entry?

Patch file itself may make sense. The composer entry - we still have to update composer file in project base and allow patching from vendor, so this seems more clear to me

Reply:
let's talk about it